src/atomic-file-codecs.ts

```diff
@@ -9,7 +9,7 @@ export const selfHealingJSONCodec = {
     return JSON.stringify(obj, null, 2);
   },
   decode: function(input: any) {
-    const str: string = input.toString();
+    const str: string = input ? input.toString() : "";
```

Comment:
Thanks for the PR! I'll try to make a release soon. Note:

```suggestion
    if (!input) return {};
```

From #4

Reply:
ah yes, that is much better